homeassistant/components/google_travel_time/translations/en.json

```diff
@@ -0,0 +1,32 @@
+{
+    "config": {
+        "abort": {
+            "already_configured": "Location is already configured"
```

Comment:
Why is this here? Isn't the key `already_configured_location`?

Reply:
See https://github.com/home-assistant/core/pull/43509/files#diff-87c7f7b9410ba2f38cad1e88e2ba0f147e2696fdbf72e3e27db8e8a6857cbf22R29 I am using `already_configured_location` because locations are being used to set the `unique_id`